macos/FluentUI/Button.swift

```diff
@@ -193,6 +193,10 @@ open class Button: NSButton {
 
 	private var hoverBackgroundColor: NSColor {
 		if #available(macOS 10.14, *) {
+			if (fillColor == .clear)
```

Comment:
Would prefer wrapping the logic in a switch statement based on the ButtonStyle, rather than checking what the fillColor is